code/modules/projectiles/projectile/reusable/arrow.dm

```diff
@@ -0,0 +1,25 @@
+/obj/item/projectile/bullet/reusable/arrow
+	name = "Wooden arrow"
+	desc = "Woosh!"
+	damage = 15
+	icon_state = "arrow"
+	ammo_type = /obj/item/ammo_casing/caseless/arrow/wood
+
+/obj/item/projectile/bullet/reusable/arrow/ash
+	name = "Ashen arrow"
+	desc = "Fire Hardened arrow."
+	damage = 25
+	ammo_type = /obj/item/ammo_casing/caseless/arrow/ash
+
+/obj/item/projectile/bullet/reusable/arrow/bone //AP for ashwalkers
+	name = "Bone arrow"
+	desc = "An arrow made from bone and sinew."
+	damage = 35
```

Comment:
I think this is a bit excessive. should be the same as the ashen arrow at most if you are gonna stack that AP on. Miners don't have as much Bullet armor as they have Melee

Reply:
yeah agreed after some tests on different armored targets.
ill make some tweaks for balance

Reply:
tweaked